pymc/distributions/distribution.py

```diff
@@ -725,6 +772,24 @@ def rv_op(
         dummy_params = [dummy_size_param, *dummy_dist_params]
         dummy_updates_dict = collect_default_updates(inputs=dummy_params, outputs=(dummy_rv,))
 
+        # Add size and updates in gufunc signature if they are missing
```

Comment:
This should be split into a helper function imo

Reply:
Done